install.sh

```diff
@@ -91,14 +91,14 @@ export MY_CONDA_DIR=$conda_prefix
 
 export PATH=$MY_CONDA_DIR/bin:$PATH
 
-echo "==== Create $env_name conda environment"
+echo "==== Create $env_name conda environment with Python 3.8"
```

Comment:
```suggestion
echo "==== Create $env_name conda environment"
```
(not sure we need to trumpet that around... it's an implementation detail)